### PR TITLE
testssl: add page

### PR DESCRIPTION
**Pull request by Rudloff**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by sbrl — APPROVED** [review]
Oh wow neat @Rudloff! Very cool indeed. I'm going to get and put this command to use myself straight away :+1:

File: pages/common/testssl.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# testssl
+
+> Check SSL/TLS protocols and ciphers supported by a server.
+> More information: <https://testssl.sh/>.
```
Comment (navarroaxel) [inline pages/common/testssl.md:4]:
```suggestion
> More information: <https://testssl.sh>.
```

Comment (Rudloff) [reply pages/common/testssl.md:4]: Is that a rule? I see both ways used in existing pages.

Technically, https://testssl.sh/ is the real URL.

Comment (navarroaxel) [reply pages/common/testssl.md:4]: You're right, the browser hides the `/` :roll_eyes:

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**